community/aircrack-ng/APKBUILD

```diff
@@ -22,25 +23,21 @@ prepare() {
 	autoreconf -fiv
 }
 
-check() {
-	cd "$builddir"
-	make check
-}
-
 build() {
 	cd "$builddir"
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
-		sqlite=true experimental=true \
```

Comment:
Why did you remove sqlite=true?

Reply:
Okay, then document it in the commit message.

Reply:
@jirutka Done.